src/components/ProjectDetails.js

```diff
@@ -0,0 +1,21 @@
+import getTask from './Task';
+
+const getProjectDetails = (tasks) => {
+  // Row for all the tasks
+  const projectsTasksRow = document.createElement('div');
+  projectsTasksRow.classList.add('row', 'project-tasks-row');
+
+  if (tasks === undefined) {
+    return [];
+  }
+
+  // Eeach task which is a card is housed in its own column
+  tasks.forEach((task) => {
+    // Getting a task as a card
+    // cardsColumn.append(getTask(task));
```

Comment:
## Remove this piece of commented out code

- [ ] `Comments`: It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))